angular-js/src/app/service/scanbot-sdk-service.ts

```diff
@@ -20,19 +20,27 @@ import {
   IBarcodeScannerHandle,
   Polygon,
   ContourDetectionResult,
-  BarcodeResult
+  BarcodeResult,
+  ITextDataScannerHandle,
+  TextDataScannerConfiguration
 } from "scanbot-web-sdk/@types";
 
 import { IMrzScannerHandle } from "scanbot-web-sdk/@types/interfaces/i-mrz-scanner-handle";
 import { MrzScannerConfiguration } from "scanbot-web-sdk/@types/model/configuration/mrz-scanner-configuration";
 import { BarcodeFormat } from "scanbot-web-sdk/@types/model/barcode/barcode-format";
 import { EngineMode } from "scanbot-web-sdk/@types/model/barcode/engine-mode";
 
+// eslint-disable-next-line import/no-webpack-loader-syntax
+require("!!file-loader?outputPath=tessdata&name=[name].[ext]!scanbot-web-sdk/bundle/bin/complete/tessdata/eng.traineddata");
+// eslint-disable-next-line import/no-webpack-loader-syntax
+require("!!file-loader?outputPath=tessdata&name=[name].[ext]!scanbot-web-sdk/bundle/bin/complete/tessdata/deu.traineddata");
+
```

Comment:
Maybe we could consider adding some kind of utility in the future, to avoid using this ugliness of a statement to import trained data 😅

Reply:
The thing is, those files should be copied into the build folder on every build. Scripting doesn't sound a good idea to me since we have to ask our customers to copy files manually by using some scripts etc. This is one of the most straightforward way to tell webpack to pack those files as well :/ It is possible to use webpack configuration file instead of ugly require statements but it means an additional configuration file for customers instead of one line solution. So I decided to go with this way until we figure out a better way :/